README.md

```diff
@@ -1,5 +1,5 @@
 <div align="center">
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <img src="logo.png" alt="logo" width="300"  height="auto" />
   <br/>
 
```

Comment:
- [ ] The License link in the README file is broken, Please make sure to fix it to make your README file more professional.